freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/config/FreeIpaConfigService.java

```diff
@@ -48,6 +49,9 @@
     @Inject
     private FreeIpaClientFactory freeIpaClientFactory;
 
+    @Inject
+    private ProxyConfigDtoService proxyConfigDtoService;
```

Comment:
it looks like to it's not used

Reply:
Finally it is used, as freeipa backup will only be executed if there is no proxy settings.